tests/run-test/extensions-shared.phpt

```diff
@@ -8,11 +10,10 @@ if (false !== stripos(`$php -n -m`, 'openssl')) {
 }
 $ext_module = ini_get('extension_dir') . DIRECTORY_SEPARATOR . (substr(PHP_OS, 0, 3) === "WIN" ? "php_openssl." : "openssl.") . PHP_SHLIB_SUFFIX;
 if( !file_exists($ext_module) ) die('skip openssl shared extension not found');
---EXTENSIONS--
-openssl
+
 --FILE--
 <?php
 var_dump(extension_loaded('openssl'));
-?>
+
```

Comment:
`?>` terminators in test files should be kept, to make tests directly runnable.

Reply:
Done, though there are a lot of existing tests missing these.

Reply:
Yeah, last year I added thousands of such terminators, but it was still not comprehensive.

Reply:
I can make run-tests warn on such tests.

Reply:
run-tests is also used by extensions, so warnings are problematic. Maybe add this to tidy.php instead.